packages/sdk-ts/src/utils/helpers.ts

```diff
@@ -20,6 +20,10 @@ export const isNode = () => {
 }
 
 export const isBrowser = () => {
+  if (isReactNative()) {
+    return false
+  }
+
```

Comment:
We have to check for this before checking if window is defined since there are cases in RN where window is polyfilled.